fs/install-to-here

```diff
@@ -33,8 +40,13 @@ until passwd; do
 	echo "Enter root password:"
 done
 
-read -p "Enter username, no spaces or big letters: "
-useradd -G sudo,video,audio,render -s /bin/bash -m $REPLY
+if [ $ROOTFS = "arch" ]; then
+  read -p "Enter username, no spaces or big letters: "
+  useradd -G video,audio,render -s /bin/bash -m $REPLY
```

Comment:
is sudo group not present in arch?

Reply:
It doesn't exist and must visudo (or similar) to provide sudo perms

Reply:
*sighs*